Proiect/new-evaluation.css

```diff
@@ -67,5 +80,64 @@
 		}
 
 		li {
-			display: inline;
+			float: left;
+			text-align: left;
+			width: 16%;
+			margin: 2px;
+			min-height: 36px;
+			font-family: Arial;
+			font-size: 15px;
+		}
+
+		#submit {
+			float: right;
+			width: 150px;
+			height: 50px;
+			background-color: #419fe2;
+			color: white;
+			border: none;
+			border-radius: 2px;
+			font-size: 20px;
+			box-shadow: 0px 3px 4px grey;
+		}
+
+		#div_footer {
```

Comment:
@zuldy mentioned this before, so I'm just going to bring it up in passing for this particular occurrence: we might want to decouple the styling from the corresponding markup. `#footer` would probably do the job just fine, and it could be safely updated to be an actual `footer` element at some later point in time, or it could be a separate `section`, or just a plain ol' `div`, all while making perfect sense and not causing any potential confusion, because the styles said one thing, while the markup said another. :)